### ISSUE TITLE
Add DataModel's read-only properties with default values

### ISSUE THREAD

**Issue by buildthomas** [LPGhatguy/lemur#102]
I am doing CI testing via lemur+testEZ for this project: https://github.com/buildthomas/MockDataStoreService. In one of my files, I need to swap between a fake datastore implementation or the real DataStoreService depending on the environment:

```lua
[...]
local shouldUseMock = false
if game.GameId == 0 then -- Local place file
	shouldUseMock = true
[...]
```

Unfortunately, it doesn't seem like lemur's Game class has any of the properties of DataModel, so I can't perform the test I want to perform on this file right now.

My suggestion is to add some properties to DataModel with sensible defaults (as if it's running in an offline Studio file):

| Read-only Property | Type | Value (static) |
|---|---|---|
| `game.CreatorId` | `int64` | `0` |
| `game.CreatorType` | `CreatorType` | `Enum.CreatorType.User` |
| `game.GameId` | `int64` | `0` |
| `game.JobId` | `string` | `""` |
| `game.PlaceId` | `int64` | `0` |
| `game.PlaceVersion` | `int` | `0` |
| `game.VIPServerId` | `string` | `""` |
| `game.VIPServerOwnerId` | `int64` | `0` |

(Properties `GearGenreSetting` and `Genre` excluded here because I don't think anyone will use these / they seem candidates for deprecation)

**Comment by LPGhatguy** [comment]
This sounds pretty straightforward! Eventually, I'd like to have an API to configure these values so you could have finer control over them.

**Comment by buildthomas** [comment]
Working on a PR!

**Comment by LPGhatguy** [comment]
Closed by #103!

@buildthomas next time you submit a PR, if you use the phrase "Closes #xxx" or "Fixes #xxx", GitHub will automatically close the related issue; it's super neat!
